cobalt/media/media_module.cc

```diff
@@ -264,6 +278,29 @@ bool MediaModule::SetConfiguration(const std::string& name, int32 value) {
   return false;
 }
 
+void MediaModule::SetPreferMinimalPostProcessingFromPersistentSettings() {
+  // Called on initialization and when the persistent setting is changed.
+  if (options_.persistent_settings == nullptr) {
+    return;
+  }
+
+  base::Value value;
+  options_.persistent_settings->Get(
+      kPreferMinimalPostProcessingPersistentSettingsKey, &value);
+  bool prefer_minimal_post_processing = value.GetIfBool().value_or(false);
+
+  const StarboardExtensionMediaSettingsApi* media_settings_api =
+      static_cast<const StarboardExtensionMediaSettingsApi*>(
+          SbSystemGetExtension(kStarboardExtensionMediaSettingsName));
+  if (media_settings_api &&
+      strcmp(media_settings_api->name, kStarboardExtensionMediaSettingsName) ==
```

Comment:
Tiny readability nitpick. I don't know if there's a styleguide somewhere, but i'd always prefer `std::string(raw_c_string) == kSomeConstant` for readability, over `strcmp(foo,bar)`